docs/web-chat/sample-integration.mdx

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 10
+title: Sample Integrations
+description: "Sample integrations in some popular frontend frameworks."
+---
+
+import { Intro } from "@site/src/components/forDocs";
+
+<Intro>
+
+Here are some sample integrations in some popular frontend frameworks to help you adhere to best practices while integrating Web Chat into your website or web app.
+
+</Intro>
+
+## React
+
+An implementation guide to integrate Web Chat using react with source code is available [here](https://github.com/helpshift/helpshift-webchat-examples).
+
+## Next.js
+
+An implementation guide to integrate Web Chat using Nextjs with source code is available [here](https://github.com/helpshift/helpshift-webchat-examples).
+
+## Angular
+
+An implementation guide to integrate Web Chat using angular with source code is available [here](https://github.com/helpshift/helpshift-webchat-examples).
```

Comment:
AngularJS

Reply:
Angular is fine since AngularJS is considered as Angular 1.0 publicly